tests/test_bloodflow.py

```diff
@@ -561,7 +561,7 @@ def test_solve_linear(point_properties, edge_properties):
     # The solution of the system is composed by a solution + a constant
     target_pressure = np.array([1.0, 0.0, -1.0])  # target solution
     result = target_pressure - pressure  # a constant
-    is_const = np.isclose(result, result[0], rtol=1e-8, atol=1e-8)
+    is_const = np.isclose(result, result[0], rtol=1e-6, atol=1e-6)
```

Comment:
I see nothing in either the PR description or commit messages that justify this. If this is necessary, this should be explained in a commit message.

Reply:
When I changed the PETSc solver, this test failed because of a small numerical error in the order of 1e-7.  For this reason, I decided to increase the tolerance.

Reply:
Fine. I think this information is worth being mentioned in the commit message of the squashed commit.

Reply:
Yes, sounds good.